modo/api.py

```diff
@@ -378,79 +379,51 @@ def enrich_metadata(self):
                 else:
                     continue
 
-    def stream_cram(
+    def stream_genomics(
         self,
-        cram_path: str,
+        file_path: str,
         region: Optional[str] = None,
         reference_filename: Optional[str] = None,
+        output_filename: Optional[str] = None,
     ):
```

Comment:
**suggestion:** would be good to add the return type hint here

Reply:
**comment:** I think the type would be  ) `Optional[Iterator[AlignedSegment | VariantRecord]]`

Reply:
Done, see https://github.com/sdsc-ordes/modo-api/commit/6633176f4e69390aa7afe4cd925b791a1a94bc9a

Reply:
Added - see https://github.com/sdsc-ordes/modo-api/commit/2f5ee25f4dce980886b32f5740d91cb9a2cbfd99, https://github.com/sdsc-ordes/modo-api/commit/cb3733b276b53600dbd2f51fed4d49cc36eb2e2a, and https://github.com/sdsc-ordes/modo-api/commit/2576f11d859baebb1a460b2f9efe5481891d4bd2